tests/foreman/destructive/test_registration.py

```diff
@@ -63,6 +64,10 @@ def test_host_registration_rex_pull_mode(
     nc = module_capsule_configured_mqtt.nailgun_smart_proxy
     module_satellite_mqtt.api.SmartProxy(id=nc.id, organization=[org]).update(['organization'])
     module_satellite_mqtt.api.SmartProxy(id=nc.id, location=[module_location]).update(['location'])
+    if module_capsule_configured_mqtt.nailgun_capsule.lifecycle_environments == []:
```

Comment:
Could this be a better catch?
```suggestion
    if module_lce not in module_capsule_configured_mqtt.nailgun_capsule.lifecycle_environments:
```
(Or we may need to iterate through id's and check the `module_lce.id` is in)

Reply:
Could work too, but as in this case, we get a capsule without LCE at the beginning of the test everytime I think this should be enough :D